src/extra/fromEvent.ts

```diff
@@ -22,18 +24,37 @@ export class DOMEventProducer implements InternalProducer<Event> {
   }
 }
 
+export class NodeEventProducer implements InternalProducer<any> {
+  public type = 'fromEvent';
+  private listener: Function;
+
+  constructor(private node: EventEmitter, private eventName: string) { }
+
+  _start(out: InternalListener<any>) {
+    this.listener = (e: any) => out._n(e);
+    const {node, eventName} = this;
```

Comment:
Because xstream is written in an extreme manner to shrink code size as much as possible, it's actually not worth doing the `const {node, eventName} = this` trick in this case, because we use `node` just once below. So we'd use just `this.node` directly. However, if we used `node` twice or more, then it'd be worth doing the trick.